tests/testthat/test_TwoStageDesign.R

```diff
@@ -89,12 +87,24 @@ test_that("plot produces correct number of columns", {
 
 
 test_that("show method", {
+  
+  expect_equal(
+    paste0(capture.output(show(design)), collapse = "\n\r"),
```

Comment:
Do you remember why there is collapse = "\n\r" here? It's probably something with Windows newlines, but its the wrong newline code. DOS newlines are "\r\n".